website/static/js/project.js

```diff
@@ -61,38 +60,8 @@ NodeActions.forkNode = function() {
                 isCors: true,
                 data: payload
             }
-        ).done(function(response) {
-            $osf.unblock();
-            bootbox.confirm({
-                message: '<h4 class="add-project-success text-success">Fork created successfully!</h4>',
-                callback: function(result) {
-                    if(result) {
-                        window.location = response.data.links.html;
-                    }
-                },
-                buttons: {
-                    confirm: {
-                        label: 'Go to new fork',
-                        className: 'btn-success'
-                    },
-                    cancel: {
-                        label: 'Keep working here'
-                    }
-                },
-                closeButton: false
-            });
-        }).fail(function(response) {
-            $osf.unblock();
-            if (response.status === 403) {
```

Comment:
By removing these checks, the user will never know why the forking failed if they don't have permission. Seems less than ideal.